Add calico support

mazdakn opened the pull request:
Add Calico support to the network policy framework. It count's the number of iptables in filter and raw table which are used for implementation for different k8s policies.

jtaleric commented on kube-burner-workload/calico/metrics.yml:
```diff
@@ -0,0 +1,154 @@
+- query: sum(irate(apiserver_request_total{apiserver="kube-apiserver",verb!="WATCH"}[2m])) by (verb,resource,code) > 0
+  metricName: APIRequestRate
+
+# Containers & pod metrics
```
Should we update the metrics to also include Calico pod names?

jtaleric commented on kube-burner-workload/calico/metrics.yml:
```diff
@@ -0,0 +1,154 @@
+
+- query: ovs_vswitchd_stream_open
+  metricName: ovsVswitchdStreamOpen
+
```
Should we add Calico specific prom queries?

mazdakn commented:
@jtaleric thanks for the initial review. I addressed your comments, and also made more changes. I can now run the whole setup without any issue. Please take another look.

npinaeva commented on kube-burner-workload/calico/README.md:
```diff
@@ -0,0 +1,24 @@
+## Running
+
+1. This profile assumes you have an openshift cluster, and the KUBECONFIG that can be used in the scale test.
```
I guess you want to search for openshift in this folder and replace with calico :)

mazdakn replied:
Thanks, Fixed it.

npinaeva commented on kube-burner-workload/calico/README.md:
```diff
@@ -0,0 +1,24 @@
+
+5. Set env variables in the `calico/env` file
+6. `source ./env`
+7. This command uses `oc` binary which is an Openshift CLI similar to kubectl
```
this command can probably refer to the new kind-metrics folder or copy some data form there to explain where  prometheus ip and ES variables are coming from

mazdakn replied:
Thanks. Fixed it. I'm going to update the README file when we have finalised the tests.

npinaeva reviewed (commented):
great to see the next platform!
If you want to include dashboards and metrics, you probably also need a local env file with ES_SERVER and ES_INDEX variables

npinaeva commented:
that lgtm, but I think we should merge https://github.com/npinaeva/k8s-netpol-scale/pull/5 and https://github.com/npinaeva/k8s-netpol-scale/pull/6 first, then rebase and test this one to ensure all timeouts and scripts work properly, wdyt?
